test/lint/test_runner/src/main.rs

```diff
@@ -429,7 +429,7 @@ fn lint_includes_build_config() -> LintResult {
                 ])
                 .args(get_pathspecs_exclude_subtrees())
                 .args([
-                    // These are exceptions which don't use bitcoin-config.h, rather the Makefile.am adds
+                    // These are exceptions which don't use bitcoin-build-config.h, rather the Makefile.am adds
```

Comment:
Not directly related to this PR change, but mentioning "Makefile.am" is outdated.